src/modules/redesign/i18n/fr_FR/avatar.json

```diff
@@ -0,0 +1,7 @@
+{
+	"current": "Current profile picture",
+	"preview": "Preview",
+	"save": "Save",
+	"select": "Select image from computer",
+	"title": "Edit profile picture"
+}
```

Comment:
translation?

Reply:
I have created all the files.
I translate them, sorry
Translation ok